app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt

```diff
@@ -200,50 +222,60 @@ private fun UserInputtingContent(
             disabledTextColor = ForzenbookTheme.colors.colors.onBackground,
             disabledPlaceholderColor = ForzenbookTheme.colors.colors.onBackground,
             disabledLabelColor = ForzenbookTheme.colors.colors.onBackground,
-            //For Icons
+            // For Icons
             disabledLeadingIconColor = MaterialTheme.colors.primary,
             disabledTrailingIconColor = MaterialTheme.colors.secondary
         ),
         singleLine = true
     )
     if (birthError && birthDate != "") {
-        Text(text = "Invalid birth date, must be 13 or older")
+        Text(text = R.string.create_account_birth_date_error.toString())
     }
-    InputField(label = "Email", value = email, onValueChange = {
-        email = it
-        emailError =
-            ((email.length > 30) || (!TextUtils.isEmpty(email) && !validateEmail(email)))
-        onTextChange(
-            firstName, lastName, birthDate, email, location,
-            firstNameError, lastNameError, birthError, emailError, locationError
-        )
-    }, imeAction = ImeAction.Next, onNext = { focusManager.moveFocus(FocusDirection.Down) })
+    InputField(
+        label = R.string.create_account_email_prompt.toString(),
+        value = email,
+        onValueChange = {
+            email = it
+            emailError =
+                ((email.length > 30) || (!TextUtils.isEmpty(email) && !validateEmail(email)))
```

Comment:
Constant

Reply:
Created a constant in the aforementioned private object.